models/Ion_auth_model.php

```diff
@@ -325,14 +327,9 @@ public function hash_password_db($id, $password, $use_sha1_override = FALSE)
 		}
 
 		// bcrypt
-		if ($use_sha1_override === FALSE && $this->hash_method == 'bcrypt')
+		if ($use_sha1_override === FALSE)
```

Comment:
We can remove the sha1 override completely right?

Reply:
I'll make another PR regarding sha1 removal. I didn't want to modify everything in one PR ;-)

Reply:
Well, actually I was wrong: this very line already remove the sha1 support!
I've updated the config file accordingly.

The PR I was talking about is more related to the `$use_sha1_override`, and by extension the way salt and token are generated. This will indeed need a dedicated PR as changes are bigger.